controller.py

```diff
@@ -280,6 +281,13 @@ def register(self, do_get=HTTP.get_with_timeout):
         if not auth_url:
             return NO_AUTH_URL
 
+        integration_contact_uri = flask.request.form.get("contact")
+        integration_contact_email = self._required_email_address(
+            integration_contact_uri, "No valid integration contact address"
```

Comment:
Unless you plan to handle this problem detail specially in the circ manager, this message should match the language used in the admin interface so it's easier for the admin to find the missing setting when they see this.